include/oboe/AudioStream.h

```diff
@@ -418,6 +418,22 @@ class AudioStream : public AudioStreamBase {
     ResultWithValue<int32_t> waitForAvailableFrames(int32_t numFrames,
                                                     int64_t timeoutNanoseconds);
 
+    /*
```

Comment:
Potentially use friends to avoid exposing these methods in public API

Reply:
Or just move to `AudioStreamBase`

Reply:
AudioStreamBuilder cannot access a protected member of a superclass on another object!
So I had to use a friend class.